packages/flowtip-react-dom/tsconfig.json

```diff
@@ -0,0 +1,20 @@
+{
+  "compilerOptions": {
+    "declaration": true,
+    "declarationDir": "./types",
+    "target": "es6",
+    "noImplicitAny": false,
+    "moduleResolution": "Node",
+    "outDir": "dist",
+    "rootDir": "./src/",
+    "sourceMap": false,
+    "strictNullChecks": true,
+    "jsx": "react"
+  },
```

Comment:
Can this be configured to be even more strict? 😛

Reply:
There appears to be:

--strict | boolean | false | Enable all strict type checking options. Enabling --strict enables --noImplicitAny, --noImplicitThis, --alwaysStrict, --strictBindCallApply, --strictNullChecks, --strictFunctionTypes and --strictPropertyInitialization.
-- | -- | -- | --

Reply:
I can turn it on and see what it does.

Reply:
Seems to be ok with `strict` on.